docpages/example_programs/the_basics/using_callback_functions.md

```diff
@@ -0,0 +1,97 @@
+\page callback-functions Using Callback Functions
+
+When you create or get an object from Discord, you send the request to its API and in return you get either an error or the object you requested/created. Let's create a message and get it:
+
+~~~~~~~~~~~~~~{.cpp}
+#include <dpp/dpp.h>
+#include <fmt/format.h> // used to format text, install and define FMT_HEADER_ONLY to use
+
+struct GetMessage {
+    dpp::snowflake message_id = 1146725845457719367ULL;
+    dpp::snowflake channel_id = 1112789762521182211ULL;
+};
+
+int main() {
+    GetMessage gm;
+
+    std::string BOT_TOKEN = "Token Was here";
+
+    /* the second argument is a bitmask of intents - i_message_content is needed to get messages */
+    dpp::cluster bot(BOT_TOKEN, dpp::i_message_content | dpp::i_default_intents);
```

Comment:
Ideally, this should be flipped to follow the same as all other pages (consistency)

Reply:
k

Reply:
For understanding, I meant flip the intents around (look at detecting_messages.md page for understanding)

Reply:
Filiped